src/PendingCalls/TestCall.php

```diff
@@ -104,6 +104,24 @@ public function throwsIf(callable|bool $condition, string|int $exception, string
         return $this;
     }
 
+    /**
+     * Asserts that the test throws the given `$exceptionClass` when called if the given condition is falsy.
```

Comment:
I think because of the casting, it's safe to say `false` here, to match the `throwsIf` method.
```suggestion
     * Asserts that the test throws the given `$exceptionClass` when called if the given condition is false.
```